### PR TITLE
Basic Parcel Metadata for Individual Counties

### PR DESCRIPTION
**Pull request by miriamseely**
Here are the metadata drafts for the Basic Parcels for each of the 29 counties. Just like the previous pull request, each county has a schema .md file and a regular metadata template .md file. As far as I could tell from the original metadata, only Kane, Rich, Utah, and Washington counties have unique fields. 

Let me know if there are corrections that need to be made! Thank you everyone!

### CODE REVIEW

File: metadata/Cadastre/basic_parcels/kane_county_parcels_schema.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+## ID
+
+0836837f-cbfc-4ae5-84f3-25befd6f9937
+
+### ZONING
+
+<!--- No definition for this field --->
+
+### ACCOUNT_NUM
```
Comment (rkelson) [inline metadata/Cadastre/basic_parcels/kane_county_parcels_schema.md:13]: I think we can remove ACCOUNT_NUM now that it has been added to all the counties and has become part of the Basic parcel schema. The Zoning field however is unique to Kane

File: metadata/Cadastre/basic_parcels/washington_county_parcels_schema.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Title
+
+Washington County Parcels Field Definitions
+
+## ID
+
+94694d7c-95d3-4bcc-a8ed-265a59ce5077
+
+### ACCOUNT_NUM
```
Comment (rkelson) [inline metadata/Cadastre/basic_parcels/washington_county_parcels_schema.md:9]: I think we can remove ACCOUNT_NUM now that it has been added to all the counties and has become part of the Basic parcel schema

File: metadata/Cadastre/basic_parcels/rich_county_parcels_schema.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Title
+
+Rich County Parcels Field Definitions
+
+## ID
+
+4757d79b-8591-4693-99b9-7e83f0205db8
+
+### NOTES
```
Comment (rkelson) [inline metadata/Cadastre/basic_parcels/rich_county_parcels_schema.md:9]: Rich does not have a unique field _NOTES_

**Review by rkelson — COMMENTED** [review]
I added some single line comments to remove _ACCOUNT_NUM_ as it it now part of the basic parcel schema. Not sure where _Notes_ came from in Rich

The only unique schemas are Kane (zoning) and Utah (ownername) as I recall

File: metadata/Cadastre/basic_parcels/rich_county_parcels_schema.md (hunk 3 above)
Comment (steveoh) [reply metadata/Cadastre/basic_parcels/rich_county_parcels_schema.md:9]: https://services1.arcgis.com/99lidPhWCzftIe9K/arcgis/rest/services/Parcels_Rich/FeatureServer/0

image

It does have that field and they are all empty.

File: metadata/Cadastre/basic_parcels/utah_county_parcels_schema.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+Utah County Parcels Field Definitions
+
+## ID
+
+2aea10f4-92ba-4539-b473-456a0675db94
+
+### OWNERNAME
+
+<!--- No definition for this field --->
```
Comment (steveoh) [inline metadata/Cadastre/basic_parcels/utah_county_parcels_schema.md:11]: Are we hoping that @rkelson provides this definition?

**Review by steveoh — COMMENTED** [review]
This looks good. Can you remind me where the data is stored that we are folding in to all of these empty headings

File: metadata/Cadastre/basic_parcels/utah_county_parcels_schema.md (hunk 4 above)
Comment (rkelson) [reply metadata/Cadastre/basic_parcels/utah_county_parcels_schema.md:11]: my bad. I was looking for Notes at the end of the fields list. Due to it's position in the list it dates back at least decade when Debby Ames requested it be there. It is no longer maintained and could be removed but that is another can of worms so we'll just leave it.

**Comment by miriamseely** [comment]
> This looks good. Can you remind me where the data is stored that we are folding in to all of these empty headings

@steveoh Here is the master doc with the field definitions for basic parcels: https://github.com/agrc/metadata-asset-tracking-tool/blob/main/metadata/Cadastre/parcels_schema.md

File: metadata/Cadastre/basic_parcels/utah_county_parcels_schema.md (hunk 4 above)
Comment (miriamseely) [reply metadata/Cadastre/basic_parcels/utah_county_parcels_schema.md:11]: Rick sent me the field definitions for the unique fields, I'll be filling them in

> Are we hoping that @rkelson provides this definition?

**Pull request merged**